test/cases/basic_test.rb

```diff
@@ -180,7 +180,7 @@ def test_point_to_json
       obj = SpatialModel.new
       assert_match(/"latlon":null/, obj.to_json)
       obj.latlon = factory.point(1.0, 2.0)
-      assert_match(/"latlon":"POINT\s\(1\s2\)"/, obj.to_json)
+      assert_match(/"latlon":"POINT\s\(1(\.0)?\s2(\.0)?\)"/, obj.to_json)
```

Comment:
Some versions return a full float when with the digit is 0.

Reply:
yes, seen this before thanks for that